src/Backend/Core/Engine/Ajax.php

```diff
@@ -36,8 +37,15 @@ public function display(): Response
 
     public function initialize(): void
     {
-        // check if the user is logged in
-        $this->validateLogin();
+        try {
+            // check if the user is logged in
+            $this->validateLogin();
+        } catch (UnauthorizedHttpException $e) {
+            $this->ajaxAction = new BackendBaseAJAXAction($this->getKernel());
+            $this->ajaxAction->output(Response::HTTP_UNAUTHORIZED, null, $e->getMessage());
```

Comment:
Als ik https://github.com/symfony/http-foundation/blob/master/Response.php#L44 dan is dat 401. Maar https://github.com/justcarakas/forkcms/blob/185544c21fde5e26615dd9515de2214e88f5ed86/src/Backend/Core/Js/backend.js#L91 verwacht een 403.

Reply:
fixed in 7e8cf405c